oarepo_vocabularies/ui/theme/assets/semantic-ui/js/oarepo_vocabularies_ui/form/components/VocabularyField/VocabularyField.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { forwardRef } from "react";
+import PropTypes from "prop-types";
+import { useFormConfig } from "@js/oarepo_ui/forms";
+import { LocalVocabularySelectField } from "../LocalVocabularySelectField/LocalVocabularySelectField";
+import { VocabularySelectField } from "../VocabularySelectField/VocabularySelectField";
+
+export const VocabularyField = forwardRef(
+  ({ fieldPath, vocabularyName, filterFunction, ...restProps }, ref) => {
+    const { formConfig } = useFormConfig();
+    const vocabularies = formConfig?.vocabularies || {};
+    const hasLocalOptions = vocabularies?.[vocabularyName]?.all?.length > 0;
```

Comment:
nitpick, could be just `const hasLocalOptions = formConfig?.vocabularies?.[vocabularyName]?.all?.length > 0;`